### DISCUSSION TITLE
Yol haritasi ne olmali.

### DISCUSSION THREAD

**Discussion by ayuksl** [dev-team-sap-react/first-react-project#1] [General]
Bekir bey ben en son trim'i de calistirdim. Bundan sonra ne yapsam iyi olur biraz hizlanmak istiyorum.

**Comment by bekircbc** [comment]
nihai hedefi belirledikten sonra iyi bir yol haritasi cikarabiliriz. ilk adimda amac calisir bir app yapmak, sonra onu debugging and refactoring ile daha da anlasilir ve iyi calisir duruma getirmek.

**Comment by bekircbc** [comment]
> Bekir bey ben en son trim'i de calistirdim. Bundan sonra ne yapsam iyi olur biraz hizlanmak istiyorum.
sizin branch i ve commitleri göremiyorum. Pull request olusturabilir misiniz?

**Comment by bekircbc** [comment]
bugün isterseniz birlikte pull request olusturalim..

**Comment by ayuksl** [comment]
Bekir bey çok iyi olur. 


Gesendet von Yahoo Mail für iPhone


Am Freitag, Oktober 13, 2023, 2:11 PM schrieb Bekir Cebeci ***@***.***>:




bugün isterseniz birlikte pull request olusturalim..
